frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -167,6 +167,7 @@ const defaultArgs: Partial<
   ComponentProps<typeof PublicOrEmbeddedDashboardView>
 > = {
   dashboard: createDashboard(),
+  downloadsEnabled: true,
```

Comment:
allow hovering the dashboard card to see the action menu, I had written loki tests for it, but it wasn't working with Loki for some reason.

![image](https://github.com/user-attachments/assets/6da4fe3c-2dee-46a7-a79f-bf30de915388)